File: demo_js/calculator2.js
function calculator(){
    const prompt=require("prompt-sync")();
    while(true){
        console.log("....Simple Calculator....");
        console.log("Enter 'x' For Exit Calculator")
        let a = parseFloat(prompt("Enter first number:"));
        let b = parseFloat(prompt("Enter second number:"));
    let op = prompt("Enter operator (+, -, *, /):");
    if (op==='x'){
        console.log("Calculator Closed...")
        break;
    }
    switch (op) {
        case '+':
            console.log("Result:"+(a+b));
            break;
        case '-':
            console.log("Result:"+(a-b));
            break;
        case '*':
            console.log("Result:"+(a*b));
            break;
        case '/':
            if (b != 0)
                console.log("Result: " + (a / b));
                else
                    console.log("Error:Cannot Divide By Zero");
            break;
        default:
            console.log("invalid choice");
        break;
        }
    }
}
calculator();

File: demo_java/calculator2.java
import java.util.Scanner;
public class calculator2 {
    public static void main(String[]args){
        System.out.println("....CALCULATOR....");
        Scanner sc=new Scanner(System.in);
        System.out.print("enter a value:");
        int a=sc.nextInt();
        System.out.print("enter b value:");
        int b=sc.nextInt();
        System.out.print("enter your choice:");
        char choice=sc.next().charAt(0);
        switch (choice) {
            case '+':
                 System.out.println("Result:"+(a+b));
                break;
            case '-':
                 System.out.println("Result:"+(a-b));
                break;
            case '*':
                 System.out.println("Result:"+(a*b));
                break;
            case '/':
                if (b != 0)
                        System.out.println("Result: " + (a / b));
                    else
                        System.out.println("Error:Cannot Divide By Zero");
                break;
            default:
                System.out.println("invalid choice");
                break;
        }
    }
}
